nxp_s32k/libuavcan/README.md

```diff
@@ -0,0 +1,51 @@
+# S32K Libuavcan V1
+### Bare-metal media layer driver for the NXP S32K14x family of automotive-grade microcontrollers, featuring CAN-FD running at 4 Mb/s and 1 Mb/s in data and nominal phases, respectively.
+An example project of it's usage for custom applications, and file dependencies used, is available in this **[Demo.](https://github.com/noxuz/libuavcan_demo)**
+| Peripheral used by this driver | Resources utilized |
+| ------------- | ------------- |
+| LPIT  | Channels 0,1 and 2, 3rd channel is available |
+| FlexCAN | All message buffers from each instance; the frame's reception interrupt handler is enabled and its priority is set to default, could be modified to meet application-specific requirements  |
+
+
+| Clocks in Normal RUN | Frequency set |
+| ------------- | ------------- |
+| CORE_CLK  | 80Mhz  |
+| SYS_CLK | 80Mhz  |
+| BUS_CLK  | 40Mhz  |
+| FLASH_CLK  | 26.67Mhz  |
+
+| Asynchronous clock sources | Divider value |
+| ------------- | ------------- |
+| SPLLDIV2  | 1  |
+
+*Asynchronous dividers not mentioned are left unset and SCG registers are locked after initial setup.*
+<br/>
+<br/>
+| Peripheral's functional clock  | Source and frequency |
+| ------------- | ------------- |
+| LPIT  | SPLLDIV2 @ 80Mhz  |
+| FlexCAN  | SYS_CLK @ 80Mhz  |
+
+| Pin location | MUX |
+| ------------- | ------------- |
+| PTE4 | CAN0 RX | 
+| PTE5 | CAN0 TX |
+| PTA12 | CAN1 RX |
+| PTA13 | CAN1 TX |
+| PTB12 | CAN2 RX |
+| PTB13 | CAN2 TX |
+| PTE10 | CAN0 STB° (GPIO) |
+| PTE11 | CAN1 STB° (GPIO) |
+
+*° UAVCAN UCANS32K146 node board-only setup, which uses the TJA1044 transceiver featuring standby (STB) mode.*
```

Comment:
Provide a quick example of how to instantiate the interface manager and obtain a driver group.

Reply:
An example like the one below and then a link to the demo would be handy or how elaborate would the example be?

`

    /* Instantiate factory object */
    libuavcan::media::S32K_InterfaceManager demo_Manager;

    /* Create pointer to Interface object */
    libuavcan::media::S32K_InterfaceGroup* demo_InterfacePtr;

    /* Initialize the node with the previously defined filtering using factory method */
    status = demo_Manager.startInterfaceGroup(&demo_Filter,Node_Filters_Count,demo_InterfacePtr);
`

Reply:
What you list here is perfect. Just put this under a heading "Creating the Interface Group"

Reply:
Added in https://github.com/noxuz/platform_specific_components/commit/9d70450643c7a2df0171b421c5814f2c9856508f